py-polars/tests/unit/testing/test_assert_series_equal.py

```diff
@@ -364,12 +366,6 @@ def test_assert_series_equal_temporal(data1: Any, data2: Any) -> None:
             {"atol": 0.1, "rtol": 0},
             id="struct_approx_equal",
         ),
-        pytest.param(
-            pl.struct(a=0, b=1.09, eager=True),
-            pl.struct(a=0, b=1, eager=True),
-            {"atol": 0.1, "rtol": 0, "check_dtype": False},
-            id="struct_approx_equal_different_type",
-        ),
```

Comment:
factoring this one out into a separate test (below)

Reply:
There was already a parametrized test for the other case (failing assert) - I moved it there.